pkg/lib/userinfo/userinfo.go

```diff
@@ -2,11 +2,24 @@ package userinfo
 
 import (
 	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	goredis "github.com/redis/go-redis/v9"
 
 	"github.com/authgear/authgear-server/pkg/api/model"
+	"github.com/authgear/authgear-server/pkg/lib/config"
+	"github.com/authgear/authgear-server/pkg/lib/infra/redis"
+	"github.com/authgear/authgear-server/pkg/lib/infra/redis/appredis"
 	"github.com/authgear/authgear-server/pkg/util/accesscontrol"
+	"github.com/authgear/authgear-server/pkg/util/duration"
 )
 
+var errCacheMiss = errors.New("cache miss")
+
+var ttl = duration.PerMinute
```

Comment:
As we will invalidate the cache I think ttl could be longer? Say 5 minutes so probably more cache hits.
A normal user usually use the app more than 1 minute.

Reply:
Are we going to make the ttl longer?

Reply:
Yes, I will make it longer. And I will add more commits to automate the load test in this PR too.